lib/reek/ast/sexp_extensions.rb

```diff
@@ -421,7 +422,7 @@ def name
         end
 
         def value
-          children.last
+          children.last if children.size > 2
```

Comment:
@troessner how would you feel about a comment something like,

```Ruby
# there are two valid forms of the casgn sexp
# (casgn <namespace> <name> <value>) and
# (casgn <namespace> <name>) used in or-asgn and mlhs
```

Here are a couple examples,
```
source = "class Hi; THIS, THAT = 2, 5; end"
Reek::Source::SourceCode.from(source).syntax_tree
(class
 (const nil :Hi) nil
 (masgn
  (mlhs
   (casgn nil :THIS)
   (casgn nil :THAT))
  (array
   (int 2)
   (int 5))))
```

```
source = "class Hi; THIS ||= 3; end"
(class
 (const nil :Hi) nil
 (or-asgn
  (casgn nil :THIS)
  (int 3)))
```